content/en/docs/release_notes/austen.md

```diff
@@ -10,7 +10,8 @@ Brief summary of the notable release contents.
 
 {{% releasenotes/upgrade %}}
 
-<!-- ## Breaking changes -->
+## Breaking changes
+* #7479: [`config.admin.updatesEnabled`](/docs/developer_guide/project/configuration/project_configuration/#admin) no longer has it's default behavior controlled by [`config.bindProcessHandlers`](/docs/developer_guide/project/configuration/project_configuration/#bindprocesshandlers). If your project unit tests still directly set `bindProcessHandlers` you should follow [this guide](/docs/updates/2022_08_26_update_unit_tests_with_test_utils) to update your unit tests to use `@axway/api-builder-test-utils` which will control `updatesEnabled` and `bindProcessHandlers` under the hood.
```

Comment:
**nitpick**: Typo

**question**: why do we mention `bindProcessHandlers`? That was just an internal trick that's been "live", for what, a month? When did we ever tell anyone to use `bindProcessHandlers`? 

```suggestion
* #7479: [`config.admin.updatesEnabled`](/docs/developer_guide/project/configuration/project_configuration/#admin) no longer has its default behavior controlled by [`config.bindProcessHandlers`](/docs/developer_guide/project/configuration/project_configuration/#bindprocesshandlers). If your project unit tests still directly set `bindProcessHandlers` you should follow [this guide](/docs/updates/2022_08_26_update_unit_tests_with_test_utils) to update your unit tests to use `@axway/api-builder-test-utils` which will control `updatesEnabled` and `bindProcessHandlers` under the hood.
```

Reply:
`bindProcessHandlers` isn't a recent internal trick, it has been explicitly in user's unit tests for years now. The change we introduced recently was to use it to control the new updates behavior.